src/schema/rules/files/raw/channels.yaml

```diff
@@ -65,23 +67,31 @@ electrodes:
     - .json
     - .tsv
   datatypes:
-    - eeg
-    - ieeg
+    - emg
   entities:
     subject: required
     session: optional
     task: optional
     acquisition: optional
     run: optional
+
+ # (i)EEG can also have `space` entity
+electrodes__eeg:
+  $ref: rules.files.raw.channels.electrodes
+  datatypes:
+    - eeg
+    - ieeg
+  entities:
+    $ref: rules.files.raw.channels.electrodes.entities
     space: optional
 
 # MEG has an additional entity available
 electrodes__meg:
-  $ref: rules.files.raw.channels.electrodes
+  $ref: rules.files.raw.channels.electrodes__eeg
```

Comment:
I am not too familiar with the BIDS standard, but does this here break backwards compatibility?

Reply:
yes, the introduction of BIDS-EMG should not have consequences for BIDS-MEG